test/test_transformers.py

```diff
@@ -104,6 +107,16 @@ def get_tolerances(
         "enable_math": False, "enable_flash": False, "enable_mem_efficient": True}
 }
 
+def get_platform_specific_sdpa():
+    if TEST_WITH_ROCM:
```

Comment:
I am think this okay to simplify the parametrization but this should use the `PLATFORM_SUPPORTS_*` variables. They are used through other places in the testing codebase and they should be updated to support rocm if thats the case at the source of truth.

Reply:
Fixed in bc97b542bf71073072de6382d90b3ed540efe24e
Now it uses `PLATFORM_SUPPORTS_FLASH_ATTENTION`/`PLATFORM_SUPPORTS_MEM_EFF_ATTENTION`